README.md

```diff
@@ -197,23 +197,27 @@ Any value in the configuration language can be a variable reference (`key=$VALUE
 
 ## Clustering
 
-NATS supports running each server in clustered mode. You can cluster servers together for high volume messaging systems and resiliency and high availability. Clients are cluster-aware. See also the [clustered NATS](http://nats.io/documentation/server/gnatsd-cluster/) documentation.
+Clustering lets you scale NATS messaging by having multiple NATS servers communicate with each other. Clustering lets messages published to one server be routed and received by a subscriber on another server. See also the [clustered NATS](http://nats.io/documentation/server/gnatsd-cluster/) documentation.
 
-The following cluster options are supported:
+### Full mesh recommended
```

Comment:
Full mesh is required right now.. We do not support directed acyclic at the moment. All servers have to be connected to each other. However, we can have the seed server set things up automatically.

/cc @kozlovic